client/src/components/common/charts/SpeedDistanceChart.tsx

```diff
@@ -0,0 +1,25 @@
+import { GREY, PURPLE } from 'components/common/charts/colours';
+import ScatterChart from 'components/common/charts/ScatterChart';
+import React from 'react';
+import { ChartProps } from 'types/chart';
+
+/**
+ * Speed-Distance chart component. Does not maintain aspect ratio.
+ *
+ * @param props Props
+ * @returns Component
+ */
+export default function SpeedDistanceChart({ data, max }: ChartProps): JSX.Element {
+  return (
+    <ScatterChart
+      title="Speed-Distance"
```

Comment:
A bit pedantic but can we have a more conventional title like 'Speed vs Distance'

Reply:
The convention from V2 is 'X-Y' so I've continued with that. It also makes it a bit easier to say. Happy to change if there's a consensus.